network/publish_test.go

```diff
@@ -359,6 +365,7 @@ func TestPublish(t *testing.T) {
 					),
 					HasCampaign: false,
 					CampaignID:  0,
+					Metadata:    []byte("{\"cli\":{\"version\":\"1\"}}"),
```

Comment:
Shouldn't it use the `metadata` variable?

```suggestion
					Metadata:    metadata,
```